cmd/snapctl/commands.go

```diff
@@ -117,6 +117,21 @@ var (
 						flRunning,
 					},
 				},
+				{
+					Name: "config",
```

Comment:
You can put plugin specific stuff inside task manifest but I think that would fall under task config. Maybe there should be a way to see that like: `snapctl task config <task-id>`?